Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.12.8-alpine
+LABEL maintainer="sberdianskyi@mate.com"
```

Comment:
There's a typo in the LABEL instruction. It should be `maintainer` instead of `mainteiner`. Please correct this to ensure proper metadata for the Docker image.